app/models/comment.rb

```diff
@@ -0,0 +1,13 @@
+class Comment < ApplicationRecord
+  # Associations
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  # Attributes
+  attribute :text, :text
+
+  # Methods
+  def update_post_comments_counter
+    post.update(comments_counter: post.comments.count + 1)
```

Comment:
- [ ] [OPTIONAL]
Similarly, `update_comment_counter` should be marked as _private_ after implementing the `after_save` method.